tests/tests.py

```diff
@@ -241,6 +242,33 @@ def test_compile_template_with_graphics_pdflatex(self):
         pdf = compile_template_to_pdf(template_name, context)
         self.assertIsNotNone(pdf)
 
+    def do_compile_escape(self, special_unicode: bool = False):
```

Comment:
This entire block can be simplified by using the `TestCase.subTest` context manager ([](https://docs.python.org/3/library/unittest.html#subtests) and `itertools.product`. I can submit a pr for that ...

Reply:
A PR would be great. Have not much experience with pure unittest -> mostly use pytest atm.

Reply:
See commits 80baed9 ff. below.

Reply:
However, ... do we really need to test how different variants of latex engines respond to unicode? The more I think about it, the more I believe that these tests are outside of the scope of `django-tex`. Plus the tests are awfully slow.

Reply:
Being a django app the user input is most likely being inputed by end users.
So actually considering also the security risks that come from crashing an application I would definitely see it in the scope of django-tex. 🤷🏻

Reply:
Another option would be that I release the escape filter itself as separate package with all the test and you just import them.
Doing so your test would only require the django + tex related stuff.

What do you think?